src/psyclone/tests/parse/kernel_test.py

```diff
@@ -133,15 +180,59 @@ def test_getkernelfilepath_caseinsensitive2(tmpdir):
     assert "tmp" in result
     assert "test_mod.f90" in result
 
-# function get_kernel_metadata
+
+# function get_kernel_interface
+def test_get_kernel_interface_match():
+    ''' Tests that something is returned when searching for an interface'''
+    module_parse_tree = parse(CODE_INTERFACE)
+    meta1, meta2 = get_kernel_interface(module_parse_tree)
+    assert meta1 is not None
+    assert meta2 is not None
+
+
+def test_get_kernel_interface_no_match():
+    ''' Tests for none when searching there is no interface'''
```

Comment:
Suggest: "...when searching a parse tree that does not contain an interface. "

Reply:
Done.